engine/installation/linux/oracle.md

```diff
@@ -144,9 +167,24 @@ Docker from the repository.
     $ sudo yum -y install docker-engine-<VERSION_STRING>
     ```
 
-    The Docker daemon starts automatically.
+    The Docker daemon does not start automatically.
+
+5.  Start the Docker daemon. Use `systemctl` on Oracle Linux 7 or `service` on
+    Oracle Linux 6.
+
+    **Oracle Linux 7**:
+
+    ```bash
+    $ sudo systemctl start docker
```

Comment:
You also need to run `systemctl enable docker` in order for Docker to start automatically on boot.